translations/README.it.md

```diff
@@ -82,4 +82,4 @@ cd React-GitHub-Resume
 
 ## 📄 Licenza
 
-Questo progetto è sotto la lincenza MIT - leggi il file [LICENSE](./LICENSE) per ulteriori dettagli
+Questo progetto è sotto la lincenza MIT - leggi il file [Licenza](./LICENSE) per ulteriori dettagli
```

Comment:
Please, correct the `LICENSE` route to `../LICENSE`

Reply:
fixed